examples/Reading-Parquet-Files-using-DuckDB.ipynb

```diff
@@ -0,0 +1,192 @@
+{
```

Comment:
<p>Ibis spins up a DuckDB connection (or whichever default backend you have) when you call something like <code>read_parquet</code>,not on import  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis-examples/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>